no-literal-qual/src/main/java/org/checkerframework/checker/noliteral/qual/MaybeDerivedFromConstant.java

```diff
@@ -0,0 +1,33 @@
+package org.checkerframework.checker.noliteral.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.DefaultFor;
+import org.checkerframework.framework.qual.LiteralKind;
+import org.checkerframework.framework.qual.QualifierForLiterals;
+import org.checkerframework.framework.qual.SubtypeOf;
+import org.checkerframework.framework.qual.TypeUseLocation;
+
+/**
+ * This type represents an expression that may have been derived (in part) from a literal, like {@code 5}, {@code "hello"},  or {@code {0xa, 0xb}}.
+
+ * This is also the type of any expression that contains a subexpression that may be a constant. For
```

Comment:
The word "also" suggests that this sentence introduces new information.  But I think this is just a consequence or an example of the rule in the first sentence.

Reply:
Does "that may be a constant" mean "of type `@MaybeDerivedFromConstant`"?  If so, say it that way.  If not, define it.